static/styles/settings.css

```diff
@@ -874,15 +874,21 @@ input[type="checkbox"] {
     }
 }
 
+.edit-bot-name {
+    margin-bottom: 20px;
+}
```

Comment:
You can use the `input-group` class instead of explicitly adding this margin.

Reply:
it wasn't working